app/javascript/util/listingUtil.ts

```diff
@@ -148,6 +149,26 @@ export const getEventTimeString = (listingEvent: ListingEvent) => {
   return ""
 }
 
+const formatEventTime = (eventTime: string) => {
+  if (eventTime) {
+    const hour = Number.parseInt(eventTime, 10)
+    const suffix = hour >= 12 ? "PM" : "AM"
+    const formattedHour = hour > 12 ? hour - 12 : hour
+    return `${formattedHour}:00 ${suffix}`
+  }
+  return ""
+}
+
+export const getEventDateTime = (eventDate: string, eventTime: string) => {
+  const startTime = eventTime?.includes(":") ? eventTime : `${formatEventTime(eventTime)}`
+  dayjs.extend(customParseFormat)
+  return dayjs(`${eventDate} ${startTime}`, "YYYY-MM-DD h:mmA")
```

Comment:
This custom format looks like it can only handle `3:00PM`, not `3:00 PM`, but it does handle both for some reason.

Reply:
Yeah that is interesting. It looks like we use strict parsing or multiple formats to be more specific: https://day.js.org/docs/en/parse/string-format#list-of-all-available-parsing-tokens